components/greaselion/browser/greaselion_service_impl.cc

```diff
@@ -0,0 +1,45 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/greaselion/browser/greaselion_service_impl.h"
+
+#include <memory>
+
+#include "brave/components/brave_rewards/common/pref_names.h"
+#include "brave/components/greaselion/browser/greaselion_download_service.h"
+
+namespace greaselion {
+
+GreaselionServiceImpl::GreaselionServiceImpl(
+    GreaselionDownloadService* download_service)
+    : download_service_(download_service) {
+  SetFeatureEnabled(REWARDS, false);
```

Comment:
I also don't think we should have to explicitly set everything to false during init. I think we just need a std::set of enabled features that starts out empty